pkg/service/types.go

```diff
@@ -0,0 +1,40 @@
+package service
+
+import (
+	"github.com/openshift/odo/pkg/odo/util/validation"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// ServiceInfo holds all important information about one service
+type ServiceInfo struct {
+	Name   string `json:"name"`
```

Comment:
This should have TypeMeta and ObjectMeta.
`Type` And `Plan` Fields should be in a separate `Spec` struct. Check `component/types.go` or `url/types.go`. We need to follow the same structure everywhere.

We can just call this strcut `Service`